app/assets/stylesheets/forever_style_guide/globals/_variables.scss

```diff
@@ -37,6 +37,11 @@ $btn-padding-large-horizontal: $padding-large-horizontal * 1.5;
 $btn-padding-small-horizontal: $padding-small-horizontal * 1.5;
 $btn-padding-xs-horizontal: $padding-xs-horizontal * 1.5;
 
+$btn-padding-default-vertical : $padding-default-vertical - 2;
```

Comment:
I realize you're following the existing pattern, but I feel like the naming convention here is incongruent w/ our typical pattern.

$btn-padding-vertical-default
$btn-padding-vertical-large

would be preferred. But I also don't think a huge refactor is in order if these variables are in use extensively in other applications.